index.html

```diff
@@ -1,96 +1,11 @@
-      icon: github
-
-    - name: Issues (Github)
-      url:  http://github.com/spring-projects/spring-security-oauth/issues
-      icon: tracking
-
-    - name: CI (Bamboo)
-      url:  https://build.springsource.org/browse/SECOAUTH-NIGHTLY
-      icon: ci
-
-#    - name: Forum
-#      url:  http://forum.spring.io/forum/spring-projects/data
-#      icon: forum
-
-    - name: Questions (StackOverflow)
-      url:  http://stackoverflow.com/questions/tagged/spring-security-oauth2
-      icon: stackoverflow
-
-#    - name: Metrics (SonarQube)
-#      url:  https://sonar.springsource.org/dashboard/index/org.springframework.data:spring-data-jpa
-#      icon: metrics
-
----
-<!DOCTYPE HTML>
+<!DOCTYPE html>
 <html lang="en-US">
-
-<!-- Specify the parent of this project (or delete if none) to influence the rendering of the breadcrumb -->
-{% capture parent_link %}
-[Spring Parent]({{ site.projects_site_url }}/spring-security)
-{% endcapture %}
-
-
-{% capture billboard_description %}
-
-Support for using Spring Security with OAuth (1a) and OAuth2.
-
-{% endcapture %}
-
-{% capture main_content %}
-
-Spring Security OAuth provides support for using Spring Security with OAuth
-(1a) and OAuth2 using standard Spring and Spring
-Security programming models and configuration idioms.
-
-## Features
-
-* Support for OAuth providers and OAuth consumers
-* Oauth 1(a) (including two-legged OAuth, a.k.a. "Signed Fetch")
-* OAuth 2.0.
-
-<span id="quick-start"></span>
-
-## Quick Start
-
-{% include download_widget.md %}
-
-Applying security to an application is not for the faint of heart, and OAuth is no exception. Before you get started,
-you're going to want to make sure you understand OAuth and the problem it's designed to address. There is good
-documentation at [the OAuth site](http://oauth.net). You will also want to make sure you understand how 
-[Spring](http://springframework.org/) and [Spring Security](http://projects.spring.io/spring-security/) work.
-
-You're going to want to be quite familiar with both [OAuth](http://oauth.net) (and/or [OAuth2](http://tools.ietf.org/html/draft-ietf-oauth-v2))
-and [Spring Security](http://projects.spring.io/spring-security/), to maximize the effectiveness of this developers guide. OAuth for
-Spring Security is tightly tied to both technologies, so the more familiar you are with them, the more likely you'll be to recognize the terminology
-and patterns that are used.
-
-
-{% endcapture %}
-
-{% capture related_resources %}
-
-### Links
-
-* [Tutorial](docs/tutorial.html)
-* [Developer's guide](docs/devguide.html)
-* [Support](docs/support.html)
-* Presentation from Spring One 2GX 2013 [Data Modelling for OAuth2](https://speakerdeck.com/dsyer/data-modelling-and-identity-management-with-oauth2)
-
-{% endcapture %}
-
-
-{% include project_page.html %}
+  <meta charset="utf-8">
+  <title>Redirecting&hellip;</title>
+  <link rel="canonical" href="https://spring.io/projects/spring-framework">
```

Comment:
These point to spring-framework instead of spring-security-oauth. I'll apply the update and merge.